demo_cli.py

```diff
@@ -53,6 +53,12 @@
     else:
         print("Using CPU for inference.\n")
     
+    ## Print helpful error message if models are not present
+    if not args.enc_model_fpath.is_file():
+        if not args.syn_model_dir.is_dir():
+            if not args.voc_model_fpath.is_file():
+                print("Error: Model files not found. If needed, download them here: https://github.com/CorentinJ/Real-Time-Voice-Cloning/wiki/Pretrained-models")
+                quit(-1)
```

Comment:
- Extract a function so you can reuse instead of duplicating the code
- Use `and`s instead of chaining `if`s
- Break the line above 100 chars

Reply:
Thank you for the suggestions @CorentinJ . The latest commit should resolve these issues. Let me know if you think of any other improvements.